e2e/app/text-labels.ts

```diff
@@ -40,6 +40,7 @@ export enum MenuOption {
   Fitbit = 'Fitbit',
   GenderIdentity = 'Gender Identity',
   LabsAndMeasurements = 'Labs and Measurements',
+  LongReadWGS = 'Long Read WGS',
```

Comment:
Added this for the new Long Read menu item. Not used currently but will most likely need it in the future.